File: 01.JS-Basic/More Exercises/PB - More Exercises/Conditional Statements/SleepyTomCat.js
function sleepyTomCat(input) {

    let hollydays = Number(input[0])
    let playNorm = 30000

    let hollydaysPlay = hollydays * 127
    let workDaysPlay = (365 - hollydays) * 63
    let totalPlay = hollydaysPlay + workDaysPlay

    if (totalPlay > playNorm) {
        console.log("Tom will run away")
        console.log(`${Math.floor((totalPlay - playNorm) / 60)} hours and ${(totalPlay - playNorm) % 60} minutes more for play`)
    }
    else {
        console.log("Tom sleeps well")
        console.log(`${Math.floor((playNorm - totalPlay) / 60)} hours and ${(playNorm - totalPlay) % 60} minutes less for play`)
    }

}
sleepyTomCat(["113"])

File: JAVA-Basic/JavaBasic/src/ConditionalStatements/ConditionalStatementsMoreExercises/SleepyTomCat.java
package ConditionalStatements.ConditionalStatementsMoreExercises;

import java.util.Scanner;

public class SleepyTomCat {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int yearHolidays = Integer.parseInt(scan.nextLine());

        int tomTargetMins = 30000;
        int workingDays = 365 - yearHolidays;
        int workingDaysPlayMins = workingDays * 63;
        int restDaysPlayMins = yearHolidays * 127;
        int totalYearPlay = workingDaysPlayMins+restDaysPlayMins;
        int differencePlay = Math.abs(tomTargetMins-totalYearPlay);

        if (totalYearPlay > tomTargetMins) {
            System.out.println("Tom will run away");
            System.out.printf("%d hours and %d minutes more for play", differencePlay / 60, differencePlay % 60);
        }else {
            System.out.println("Tom sleeps well");
            System.out.printf("%d hours and %d minutes less for play", differencePlay / 60, differencePlay % 60);
        }
    }
}
